csharp-basics/exercises/Loops/Loops/Exercise9/RollTwoDice.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Exercise9
+{
+    internal class RollTwoDice
+    {
+        static void Main(string[] args)
+        {
+            Random random = new Random();
+
+            Console.WriteLine("Welcome to game RollTwoDice. Press enter if you want to start the game");
+            Console.ReadKey();
+
+            Console.Write("Enter a number of your desired sum from 2 - 12: ");
+            var desiredSum = Convert.ToInt32(Console.ReadLine());
+
+            for (int i = 0; i < 1000; i++)
```

Comment:
Kāpēc šāds cipars 1000?  "until the sum of the two dice values is the desired sum"